scripts/language_model/cache_language_model.rst

```diff
@@ -0,0 +1,65 @@
+Cache Language Model
+---------------------
```

Comment:
This should be listed as a section in word language model, as they share the same scripts package. It's currently completely separate. http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-110/40/scripts/index.html#cache-language-model